<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_new">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andy</family>
      <pointsize>-1</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24px &quot;Andy&quot;;</string>
    </property>
    <property name="text">
     <string>New Puzzle</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_difficulty">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18px &quot;Andy&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Very Hard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eazy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Very Eazy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eazy AF</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_solve">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>460</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24px &quot;Andy&quot;;</string>
    </property>
    <property name="text">
     <string>Auto Solve</string>
    </property>
   </widget>
   <widget class="QLabel" name="player_status">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>380</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(208, 208, 208);
font: 75 26px &quot;Andy&quot;;</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 26px &quot;Andy&quot;;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_hint">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 26px &quot;Andy&quot;;</string>
    </property>
    <property name="text">
     <string>Hint</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ans">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>530</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18px &quot;Andy&quot;;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_note">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameTimeLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(208, 208, 208);
font: 75 26px &quot;Andy&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
